spec/views/users/_procedure_footer.html.haml_spec.rb

```diff
@@ -27,4 +27,47 @@
     it { is_expected.to have_link("Accessibilité") }
     it { is_expected.not_to have_text('téléphone') }
   end
+
+  describe '#cadre_juridique' do
+    context 'quand c\'est un lien texte' do
+      before { dossier.procedure.update(cadre_juridique: "http://google.fr") }
+      it { is_expected.to have_link("Texte cadrant la demande d'information") }
+      it { is_expected.to have_selector('a[href="http://google.fr"]') }
```

Comment:
Pinaillage : les tests de démarche sont assez long à mettre en place (parce que créer une Procedure de zéro, c'est long). Du coup c'est bien si on évite de créer deux fois l'objet, en regroupant les specs :

```suggestion
      it { is_expected.to have_link("Texte cadrant la demande d'information", href: http://google.fr) }
```